ui/app/routes/vault/cluster/secrets/backend/configuration.js

```diff
@@ -6,8 +6,9 @@
 import { service } from '@ember/service';
 import Route from '@ember/routing/route';
 
-export default Route.extend({
-  store: service(),
+export default class SecretsBackendConfigurationRoute extends Route {
+  @service store;
+
```

Comment:
Thanks for updating this router file! I wonder if we should update the syntax below everywhere it says `backend.set` to `backend.casRequired = backend.casRequired ? backend.casRequired : 'False'`. Not necessary but it might be nice cleanup.

I'll defer to you!